### DISCUSSION TITLE
Extra hard break is added in editor, but not in returned html

### DISCUSSION THREAD

**Discussion by jameskzhao** [ueberdosis/tiptap#2538] [General Questions]
Hi, I am working on a project with tiptap and found the editor automatically append a `<br>` tag in the following HTML structure
```
<p>
some text
<br>
</p>
```
So it becomes 

```
<p>
some text
<br>
<br>
</p>
```

in the editor view. 

However when I call `editor.getHtml()`, the 2nd `<br>` tag is not available. 

My users ended up complaining that their content looks correct in editor view, but missing a new line on the final page. How can I fix this problem?

**Comment by andredewaard** [comment]
found a solution for this?

**Reply by andredewaard** [reply]
Solution https://tiptap.dev/api/nodes/hard-break
